internal/dependency-track/config/application.properties

```diff
@@ -0,0 +1,222 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we add some comment to the configuration values we have altered, in comparison to original values. Also add a small clarification about each change. Example: https://github.com/wso2/carbon-kernel/blob/f2a1eaf9c1fa2847ebff4d0c8932096dda4371e9/distribution/kernel/carbon-home/repository/conf/security/Owasp.CsrfGuard.Carbon.properties#L104

Reply:
Done